src/types/definitions.d.ts

```diff
@@ -8,3 +9,39 @@ export type EventConstructor<T extends DiscordEvent> = new (
 export type InteractionConstructor<T extends Interaction> = new (
     client: DiscordClient,
 ) => T;
+
+export type LoopMode = 'off' | 'current' | 'all';
```

Comment:
do you have a reason for using a union type instead of an enum?

Reply:
damn you right